dali/base/dasess.cpp

```diff
@@ -557,35 +565,8 @@ class CSessionRequestServer: public Thread
                 const SocketEndpoint &peerIP = coven.queryComm().queryChannelPeerEndpoint(mb.getSender());
                 Owned<INode> servernode(deserializeINode(mb));  // hopefully me, but not if forwarded
                 int role=0;
-                if (mb.length()-mb.getPos()>=sizeof(role)) { // a capability block present
+                if (mb.length()-mb.getPos()>=sizeof(role)) // a capability block present
                     mb.read(role);
-                    if (!manager.authorizeConnection(peerIP, (DaliClientRole) role))
```

Comment:
What happens if the client does not call initClientProcess?  Will it fail on the first connection?  E.g. some legacy clients.

Reply:
the whitelist when it connects (ultimately MP does via verifyConnection, that calls connect()).

The role is supplied via MP start up with this change, legacy clients (i.e. pre-7.0.x) will not supply a role and so the 'role' can't be screen for, with the default settings (allowAnonRoles) it will verify using IP only. NB: This is covered in my notes in 1st message of this PR.

This the same as clients that didn't call initClientProcess before, which happens for example when a foreign components makes direct Dali connections.